.travis.yml

```diff
@@ -3,9 +3,5 @@ language: ruby
 rvm:
   - 2.1.4
 
-branches:
-  only:
-    - develop
```

Comment:
I want to start from scratch following Travis' docs. Actually I don't remember what this means.